cypress/integration/advanced-functionalities/admin/system/users/Delete.spec.js

```diff
@@ -13,34 +13,31 @@ describe('Test for deleting a user', () => {
   context('Test for deleting a user', () => {
     it('should be able to delete a user', () => {
       cy.visit(adminURL + '/')
-      // We wait for 2s in order the page to be fully loaded
-      cy.wait(2000)
+      // We wait for 3s in order the page to be fully loaded
+      cy.wait(3000)
       cy.get('.nav-sidebar').contains('Users').click()
-      // We wait 1s in order the page to be fully loaded
+      // We wait 2s in order the page to be fully loaded
+      cy.wait(2000)
+      cy.get('[data-test-id="input-search"]').type('Permissions')
+      // We wait 1s for the search to finish
       cy.wait(1000)
       cy.contains('Permissions account').get('#resource-list > tbody > tr:last > td:last > a').click()
       cy.get('[data-test-id="button-delete"]').click()
       cy.get('.confirmation-confirm').click()
       // We confirm that the action was completed successfully
       cy.get('.b-toast-success')
-    })
-  })
-
-  context('Test for un-deleting a user', () => {
-    it('should be able to un-delete a user', () => {
       cy.get('[data-test-id="filter-deleted-users"]').contains('Only').click()
```

Comment:
let's create a separate folder called filters and add two files - one for deleted and another for suspended; this test is too big